lib/methods/images.ts

```diff
@@ -246,7 +246,8 @@ export async function executeImageCompare(
 
   // 5.		Execute the compare and retrieve the data
   const data: CompareData = await compareImages(readFileSync(baselineFilePath), readFileSync(actualFilePath), compareOptions);
-  const misMatchPercentage = imageCompareOptions.rawMisMatchPercentage
+  const misMatchPercentage = data.rawMisMatchPercentage;
```

Comment:
When we're touching this code, can we make it more clear by changing the name of the const? We then also need to change it on multiple places, including line 254/255

```suggestion
  const rawMisMatchPercentage = data.rawMisMatchPercentage;
```